--- a/src/gba/dma.c
+++ b/src/gba/dma.c
@@ -129,11 +129,13 @@
 		}
 #ifndef __LIBRETRO__
 		mLOG(GBA_DMA, INFO, "Starting DMA %i 0x%08X -> 0x%08X (%04X:%04X)", dma,
+		     currentDma->nextSource, currentDma->nextDest,
+		     currentDma->reg, currentDma->count & 0xFFFF);
 #else
 		mLOG(GBA_DMA, DEBUG, "Starting DMA %i 0x%08X -> 0x%08X (%04X:%04X)", dma,
-#endif
 		     currentDma->nextSource, currentDma->nextDest,
 		     currentDma->reg, currentDma->count & 0xFFFF);
+#endif
 
 		currentDma->nextSource &= -width;
 		currentDma->nextDest &= -width;
